server/konnectors/edf.coffee

```diff
@@ -0,0 +1,1347 @@
+xml2js = require 'xml2js'
+request = require 'request'
+async = require 'async'
+moment = require 'moment'
+cozydb = require 'cozydb'
+File = require '../models/file'
+
+parser = new xml2js.Parser()
+builder = new xml2js.Builder headless: true
+
+# TODO :
+# Move updateOrCreate in a lib
+# multi account
+# multi contract
+
+# Models
+Client = cozydb.getModel 'Client',
+    clientId: String
+    vendor: String
+    numeroAcc: String
+    address: Object
+    name: Object
+    email: String
+    telMobile: String
+    telHome: String
+    loginEmail: String
+    coHolder: Object
+    commercialContact: Object
+    docTypeVersion: String
+
+Contract = cozydb.getModel 'Contract',
+    clientId: String
+    vendor: String
+    number: String
+    start: String
+    end: String
+    status: String
+    terminationGrounds: String
+    pdl: String
+    energie: String
+    nomOffre: String
+    numeroDepannage: String
+    puissance: String
+    structureTarifaire: String
+    optionPrix: String
+    compteur: Object
+    consommationAnnuelle: Number
+    horrairesHeureCreuses: String
+    releve: Object
+    services: [Object]
+    docTypeVersion: String
+
+PaymentTerms = cozydb.getModel 'PaymentTerms',
+    vendor: String
+    clientId: String
+    bankDetails: Object
+    solde: Number
+    typeEncaissement: String
+    modifIbanAutorisee: Boolean
+    dernierReglement: Object
+    periodiciteFacture: String
+    dateProchaineFacture: String
+    idPayeur: String
+    payeurDivergent: Boolean
+    paymentSchedules: [Object]
+    docTypeVersion: String
+
+Home = cozydb.getModel 'Home',
+    pdl: String
+    beginTs: String
+    isProfileValidated: Boolean
+    housingType: String
+    residenceType: String
+    occupationType: String
+    constructionDate: String
+    isBBC: Boolean
+    surface: Number
+    occupantsCount: Number
+    principalHeatingSystemType: String
+    sanitoryHotWaterType: String
+    docTypeVersion: String
+
+ConsumptionStatement = cozydb.getModel 'ConsumptionStatement',
```

Comment:
Yes. But, as with Cozy, access control is based on model ; I think it's better to have a different model for phone call log and energy consumption . Because I don't want that apps the app which offers some services around energy have access to my phone call log ; end the phone number of all my correspondents.